src/api/app/assets/javascripts/webui/application.js.erb

```diff
@@ -28,7 +28,7 @@
 //= require peek
 //= require rails-timeago
 //
-//= require webui/keybindings.js.coffee
+//= require webui/keybindings.js
```

Comment:
@DavidKang you are removing the extension here, but not on the file it self. Is that by purpose?

Reply:
@krauselukas, it was on purpose, because there is no need to include all the pre-processor extensions in a manifest.